release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -5193,3 +5193,101 @@ openshift3/ose-template-service-broker:v3.11.521-1.git.59aef03
 ==== Upgrading
 
 To upgrade an existing {product-title} 3.10 or 3.11 cluster to this latest release, see xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrade methods and strategies] for instructions.
+
+[[ocp-3-11-523]]
+
```

Comment:
Chis, this is nitpicky, but do you mind removing the blank lines between the heading ID and the heading?  My comment is related to consistency and convention, but if there's a technical reason for the blank line, just say so.

Reply:
Removed extra spaces there in an a few other locations.